lib/plugins/inspec-parallel/lib/inspec-parallel/runner.rb

```diff
@@ -24,6 +24,8 @@ def run
         initiate_background_run if run_in_background # running a process as daemon changes parent process pid
         until invocations.empty? && @child_tracker.empty?
           while should_start_more_jobs?
+            # Changing output to STDERR to avoid the output interruption between runs
+            ChefLicensing::Config.output = STDERR
```

Comment:
Could we reset back to it's original output stream after this `while` logic is completed?

Ignore if such a case doesn't arise where there's execution of code post this `while` block.

Reply:
And it could be set before the `while` loop starts. Thoughts?
Something like
```
original_stdout_stream = ChefLicensing::Config.output
ChefLicensing::Config.output = STDERR

while do
# something
end

ChefLicensing::Config.output = original_stdout_stream

```

Reply:
> Could we reset back to it's original output stream after this `while` logic is completed?
> 
> Ignore if such a case doesn't arise where there's execution of code post this `while` block.

It seems safe to do so. In case anyone runs `inspec parallel` parallelly using a script or something. However, that would be a bad use of `inspec parallel` command. (Running parallel command parallelly)
Otherwise, for a single invocation, I feel we do not need to reset as such. We can discuss this.

Reply:
> And it could be set before the `while` loop or the `until` loop starts. Thoughts? Something like
> 
> ```
> original_stdout_stream = ChefLicensing::Config.output
> ChefLicensing::Config.output = STDERR
> 
> while do
> # something
> end
> 
> ChefLicensing::Config.output = original_stdout_stream
> ```

Yes, this makes sense. Initially, I intended to have a check that this should be set only the first time any process is forked or spawned. But later removed that check. 
Makes more sense to put it before the while condition.

Reply:
> Could we reset back to it's original output stream after this `while` logic is completed?
> 
> Ignore if such a case doesn't arise where there's execution of code post this `while` block.

Also I would not want to reset it after every while loop. I would rather do it after until, if at all.